examples/consume_recover_retry.py

```diff
@@ -0,0 +1,35 @@
+import functools
+import pika
+from retry import retry
+
```

Comment:
pls rename the .py file to match the corresponding .rst file to make it clear that this is for blocking connections and to make it easier for developers to correlate the two.

Also see feedback in corresponding .rst.